### PR TITLE
Fix all but one type error

### PR DESCRIPTION
**Pull request by auscompgeek**

### CODE REVIEW

**Comment by auscompgeek** [comment]
In case you're curious as to what the reviewdog github-check output looks like: https://github.com/auscompgeek/robotpy-installer/runs/1975777819?check_suite_focus=true

**Review by virtuald — APPROVED** [review]
LGTM, have you actually tested this?

File: .github/workflows/reviewdog.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install -e .
+      - uses: tsuyoshicho/action-mypy@v3
+        with:
+          reporter: github-check
+          target: robotpy_installer
+          mypy_flags: --ignore-missing-imports
```
Comment (virtuald) [inline .github/workflows/reviewdog.yml:19]: Can't you add a `#type: ignore` comment to the pip import instead?

Comment (auscompgeek) [reply .github/workflows/reviewdog.yml:19]: That's probably actually a better idea.

**Comment by auscompgeek** [comment]
> LGTM, have you actually tested this?

This only fixes one real (i.e. not broken type hints) bug: `opkg install -r ...` was broken. `download -r requirements.txt` is still broken. I won't have access to a roboRIO until Saturday (and I don't have the disk space for a VM either).

**Comment by auscompgeek** [comment]
Well, `robotpy-installer install -r requirements.txt` works.

**Pull request merged**